### DISCUSSION TITLE
Waterfall Timestamping

### DISCUSSION THREAD

**Discussion by fubar-life123** [AlexandreRouma/SDRPlusPlus#347] [Ideas]
This SDR is great! Only thing missing for me is Timestamps in the waterfall display!

**Comment by richgroot** [comment]
What would they look like, in your view?  When are they useful?

**Comment by fubar-life123** [comment]
Identify radio devices by poling interval. Most devices (wireless
thermometers as an example) transmit at fixed intervals.
